src/server/src/test/java/io/cassandrareaper/service/SegmentRunnerTest.java

```diff
@@ -277,6 +274,141 @@ protected JmxProxy connect(final Optional<RepairStatusHandler> handler, String h
     assertEquals(0, storage.getRepairSegment(runId, segmentId).get().getFailCount());
   }
 
+  @Test
+  public void disorderedNotificationsTest()
```

Comment:
this is failing a lot in travis tests.

for example:
 - https://travis-ci.org/thelastpickle/cassandra-reaper/jobs/314826143
 - https://travis-ci.org/thelastpickle/cassandra-reaper/jobs/314825786